aws/solutions/AmazonCloudWatchAgent/inline/amazon_linux.template

```diff
@@ -131,7 +141,11 @@ Resources:
         # This script below is to install AmazonCloudWatchAgent, restart AmazonCloudWatchAgent and tell the result to cloudformation.
         Fn::Base64: !Sub |
            #!/bin/bash
-           rpm -Uvh https://s3.amazonaws.com/amazoncloudwatch-agent/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm
+           curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+           unzip awscliv2.zip
+           sudo ./aws/install
+           aws s3 cp ${PackageLocation} .
+           rpm -Uvh amazon-cloudwatch-agent.rpm
```

Comment:
Instead of using the AWS CLI to copy the S3 file, why don't we just update the URL for the rpm call to use the stack parameter?
```suggestion
           rpm -Uvh ${PackageLocation}
```